packages/storybook-vue/stories/components/modal/Modal.stories.mdx

```diff
@@ -43,6 +43,13 @@ import ScaleModal from './ScaleModal.vue';
         disable: true,
       },
     },
+    scaleBeforeClose: {
+      description: '@deprecated in v3 in favor of kebab-case event names.',
+    },
+    'scale-before-close': {
+      description:
+        'It fires on every close attempt. Calling `event.preventDefault()` will prevent the modal from closing.',
```

Comment:
Would it be possible to add a bit more info? something like this:

`event.detail.trigger` can be one of `CLOSE_BUTTON`, `ESCAPE_KEY` or `BACKDROP`, depending on the action that triggered the event.

Reply:
updated.

Reply:
Thank you!